enhancements/proposals/multi-datacenter.md

```diff
@@ -0,0 +1,152 @@
+# Multi-datacenter support
+
+## Summary
+
+With the current state of the operator, it is impossible to create a multi-datacenter installation in a simple enough way.
+This proposal introduces the changes necessary to make it possible to manually connect multiple clusters across different datacenters.
+Additionally, while it is now impossible to expose the cluster externally, the document proposes different levels of exposure and an interface for doing so. TODO
+
+## Motivation
+
+TODO
+
+### Goals
+
+* Provide multi-datacenter support
+* Introduce different cluster  exposure types
```

Comment:
Given the exposure types are needed for standalone clusters as well, would it be better to have a dedicated proposal?